piqueserver/config.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2017 Piqueserver development team
+
+# This file is part of piqueserver.
+
+# piqueserver is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# piqueserver is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with piqueserver.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import toml
+
+
+DEFAULT_STYLE = 'TOML'
+TOML_STYLE = 'TOML'
+JSON_STYLE = 'JSON'
+
+
+class ConfigException(Exception):
```

Comment:
Do we really need a custom Exception class for this? The only point of custom exceptions is catching only a more narrow case, or giving additional information to the catchee, but I don't see how this would help here... If we need this, we should subclass ValueError though.